.github/workflows/deploy-tombstone.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy Tombstone
+
+on:
+  pull_request:
+    paths:
+      - "tombstone/**"
+  push:
+    branches:
+      - main
+    paths:
+      - "tombstone/**"
+
+  workflow_dispatch: {}
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
+
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v1
+        with:
+          path: "./tombstone"
+
+  deploy:
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    steps:
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v2
```

Comment:
Also switching to the new method of deploying to GH pages, where you build and upload a zip of the site, rather than committing to a special branch.